lib/manager/flux/extract.spec.ts

```diff
@@ -0,0 +1,187 @@
+import { loadFixture } from '../../../test/util';
+import { GlobalConfig } from '../../config/global';
+import type { RepoGlobalConfig } from '../../config/types';
+import { SkipReason } from '../../types';
+import type { ExtractConfig } from '../types';
+import { extractAllPackageFiles, extractPackageFile } from './extract';
+
+const config: ExtractConfig = {};
+const adminConfig: RepoGlobalConfig = { localDir: '' };
+
+describe('manager/flux/extract', () => {
+  beforeEach(() => {
+    GlobalConfig.set(adminConfig);
+  });
+
+  afterEach(() => {
+    GlobalConfig.reset();
+  });
+
```

Comment:
```suggestion
```
Not required, as the `set` will do it also.

Reply:
OK, I had copied this from some of the `gitlabci` manager tests. I'll remove it.